src/app/domain/backup/services/manageRemoteBackupConfig.ts

```diff
@@ -233,10 +238,25 @@ export const createRemoteBackupFolder = async (
   return remoteBackupConfig
 }
 
+const isFileCorrespondingToMedia = (file: File, media: Media): boolean => {
+  return (
+    file.name === media.name &&
+    new Date(file.created_at).getTime() === media.creationDate &&
+    new Date(file.updated_at).getTime() === media.modificationDate
+  )
+}
```

Comment:
Is it resilient to time zone changes? aka: I took lots of photos while on vacation in Hawaii, and I don't do backups during that time. Or only a part of it. And then I go back to France. What happens then?

Reply:
Original date comes from react-native-cameraroll which is an UTC timestamp. It is then sent to the stack with toISOString() which is also always on UTC time zone. And getTime return again the UTC timestamp from the stack date. So it should be okay!

Reply:
Great to hear!